rdr_service/offline/main.py

```diff
@@ -902,6 +910,12 @@ def _build_pipeline_app():
         view_func=reconcile_gc_data_file_to_table,
         methods=["GET"]
     )
+    offline_app.add_url_rule(
+        OFFLINE_PREFIX + "ReconcileRawToAw1Ingested",
+        endpoint="reconcile_raw_to_aw1_ingested",
+        view_func=reconcile_raw_to_aw1_ingested,
+        methods=["GET"]
+    )
```

Comment:
Do you need to add this to the `cron` yaml files for the schedule?

Reply:
This has been added.